dss-builder/rootfs.skel/usr/bin/dss-deploy-pilot

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python3
+"""
+Service deploy agent.
+
+This agent is installed on the builder instance, which is configured and launched by scripts/launch_dss_builder.sh.
+
+When the instance starts, the agent is launched by the systemd service configured by
+dss-builder/rootfs.skel/etc/systemd/system/multi-user.target.wants/dss-deploy-pilot-dev.service
+(/etc/systemd/system/multi-user.target.wants/dss-deploy-pilot-dev.service on the instance).
+
+The agent polls GitHub for new deployment objects, which are created by running "make .
+"""
+
+import os, sys, subprocess, json, logging, argparse, shutil, signal, socket, time
+import boto3, requests
+from datetime import datetime
+
+def get_metadata(path):
+    return requests.get("http://169.254.169.254/latest/meta-data/{}".format(path)).content.decode()
```

Comment:
Why is the IP address hardcoded?

Reply:
https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-metadata.html